packages/nextjs-kit/src/__tests__/snapshotTests/paginator.test.tsx

```diff
@@ -83,66 +57,126 @@ const RenderCurrentItems: React.FC<PaginationItemProps> = ({
+          </section>
+        </main>
+      </div>
     );
     expect(
-      (document.getElementById('back-btn') as HTMLInputElement).disabled
+      (document.getElementById('back-btn') as HTMLButtonElement).disabled
     ).toBe(true);
   });
 
   it('next button is disabled on last page', () => {
     render(
-      <TestComponentToRender
-        breakpoints={{ start: 6, end: 12, add: 6 }}
-        itemsPerPage={60}
-      />
+      <div className="prose container min-w-full min-h-screen max-w-screen mx-auto">
+        <main className="flex mx-auto flex-col">
+          <section className="mx-auto">
+            <h1 className="my-10">Pagination example</h1>
+            <Paginator
+              data={examplePaginationData}
+              itemsPerPage={60}
+              breakpoints={{ start: 6, end: 12, add: 6 }}
+              routing={true}
+              Component={RenderCurrentItems}
+            />
+          </section>
+        </main>
+      </div>
     );
     fireEvent.click(screen.getByText('>'));
     fireEvent.click(screen.getByText('>'));
     expect(
-      (document.getElementById('next-btn') as HTMLInputElement).disabled
+      (document.getElementById('next-btn') as HTMLButtonElement).disabled
     ).toBe(true);
   });
 
   it('check breakpoints', () => {
     render(
-      <TestComponentToRender
-        breakpoints={{ start: 2, end: 12, add: 6 }}
-        itemsPerPage={10}
-      />
+      <div className="prose container min-w-full min-h-screen max-w-screen mx-auto">
+        <main className="flex mx-auto flex-col">
+          <section className="mx-auto">
+            <h1 className="my-10">Pagination example</h1>
+            <Paginator
+              data={examplePaginationData}
+              itemsPerPage={10}
+              breakpoints={{ start: 2, end: 12, add: 6 }}
+              routing={true}
+              Component={RenderCurrentItems}
+            />
+          </section>
+        </main>
+      </div>
     );
     expect(screen.queryAllByText('2')).toHaveLength(0);
     fireEvent.click(screen.getByText('...'));
     expect(screen.queryAllByText('2')).toHaveLength(1);
   });
   it('check next button works', () => {
     const { asFragment } = render(
-      <TestComponentToRender
-        breakpoints={{ start: 6, end: 12, add: 6 }}
-        itemsPerPage={10}
-      />
+      <div className="prose container min-w-full min-h-screen max-w-screen mx-auto">
+        <main className="flex mx-auto flex-col">
+          <section className="mx-auto">
+            <h1 className="my-10">Pagination example</h1>
+            <Paginator
+              data={examplePaginationData}
+              itemsPerPage={10}
+              breakpoints={{ start: 6, end: 12, add: 6 }}
+              routing={true}
+              Component={RenderCurrentItems}
+            />
+          </section>
+        </main>
+      </div>
     );
     fireEvent.click(screen.getByText('>'));
     expect(asFragment()).toMatchSnapshot();
```

Comment:
I would expect this to fail. If the next button was hit we would need to re-render the snapshot and see what is on page 2.

It might also be worth [adding a `spyOn`](https://vitest.dev/api/#vi-spyon)to the router mock to see if `push` was called, that way if routing breaks in the component the tests will let us know.